libmachine/provision/os_release.go

```diff
@@ -69,7 +69,7 @@ func (osr *OsRelease) ParseOsRelease(osReleaseContents []byte) error {
 	for scanner.Scan() {
 		key, val, err := parseLine(scanner.Text())
 		if err != nil {
-			return err
+			log.Warn("Warning: got an invalid line error parsing /etc/os-release: %s", err)
```

Comment:
Also - `continue`ing will also avoid an error I ran into recently in #1449 where `setIfPossible` barfed because of an extra blank line in `/etc/os-release`.